backend-diagram-converter/core/src/main/java/org/camunda/community/converter/ExpressionUtil.java

```diff
@@ -1,47 +1,80 @@
 package org.camunda.community.converter;
 
-import java.util.Optional;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import org.camunda.community.converter.BpmnDiagramCheckResult.Severity;
 
 public class ExpressionUtil {
   private static final ExpressionUtil INSTANCE = new ExpressionUtil();
 
   private ExpressionUtil() {}
 
-  public static Optional<String> transform(String possibleExpression, boolean createExpression) {
-    if (possibleExpression == null) {
-      return Optional.empty();
+  public static String transform(final String expression, DomElementVisitorContext context) {
+    String transform = INSTANCE.doTransform(expression);
+    if (context != null) {
+      context.addMessage(
+          Severity.TASK,
+          "Please review transformed expression: '" + expression + "' -> '" + transform + "'");
     }
```

Comment:
This nearly doubles the amount of tasks, as most transformation include a feel expression as well:
```
- TASK: Please review transformed expression: 'management' -> 'management'
- TASK: Attribute 'candidateGroups' on 'userTask' was mapped, please review. Hint: Please review candidate groups 'management'
```
Could we merge this into a single Task?
Like 
```
- TASK: Attribute 'candidateGroups' on 'userTask' was mapped, please review. Hint: Please review candidate groups with new expression `management`
```

Reply:
This is better now.